LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2017 NAXAM CO., LTD
```

Comment:
I appreciate using MIT license, but considering this is a pull request you shouldn't attribute to yourself.
Please add mikescandy

Reply:
Alright, let me update. It's our default template.